src/components/Button/Button.scss

```diff
@@ -150,15 +150,15 @@ $stacking-order: (
 }
 
 .outline {
-  @include button-outline(color('ink', 'lighter'));
```

Comment:
The `$outline-color` parameter doesn't seem to be used anymore. https://github.com/Shopify/polaris-react/blob/main/src/styles/shared/_buttons.scss#L118

Perhaps we can just remove passing in the argument entirely: `@include button-outline()`